site/content/en/reference/live/alpha.md

```diff
@@ -0,0 +1,156 @@
+-->
+
+### How to Upgrade to the Next Generation Inventory Object (Alpha)
+
+#### What is an Inventory Object
+
+An inventory object is the automatically generated object which keeps track
+of the set of objects applied together. The current inventory object type
+is a [**ConfigMap**](https://kubernetes.io/docs/concepts/configuration/configmap/),
+and it is usually defined in a package file called
+**inventory-template.yaml**. This file is created from an invocation of
+`kpt live init`. A typical use of the inventory object is to prune (delete)
+objects omitted locally.
+
+#### What is a Next Generation Inventory Object
+
+The next generation inventory object is a **ResourceGroup** custom resource
+replacing the current [**ConfigMap**](https://kubernetes.io/docs/concepts/configuration/configmap/).
+Because the new inventory object is a
+custom resource, you must have permissions to add a custom resource
+definition (CRD) to the cluster. The individual command to add the
+**ResourceGroup** CRD is:
+
+```sh
+export RESOURCE_GROUP_INVENTORY=1
+kpt live install-resource-group
+```
+
+#### Upgrade Scenario 1: New (Uninitialized) Packages
+
+Packages which are newly downloaded and uninitialized should follow the
+following steps:
+
+1. `export RESOURCE_GROUP_INVENTORY=1`
+2. `kpt live init <PACKAGE DIR>`
+
+The `init` step adds an `inventory` section in the
+package Kptfile. The package is now eligible to be applied
+with `kpt live apply <PACKAGE DIR>`. This command will
+automatically apply the **ResourceGroup** CRD if it has not
+already been applied.
+
+#### Upgrade Scenario 2: Existing (Initialized) Packages
+
+Existing packages which have already been initialized, should follow
+the following steps:
+
+1. `export RESOURCE_GROUP_INVENTORY=1`
+2. `kpt live migrate <PACKAGE DIR>`
+
+Initially, the `migrate` command applies the **ResourceGroup** CRD.
+Then the `migrate` command replaces the
+[**ConfigMap**](https://kubernetes.io/docs/concepts/configuration/configmap/)
+inventory object in the cluster (if it exists) with a **ResourceGroup**
+custom resource. The `migrate` command also deletes the local inventory
+[**ConfigMap**](https://kubernetes.io/docs/concepts/configuration/configmap/)
+config file (usually **inventory-template.yaml**). If this local
+[**ConfigMap**](https://kubernetes.io/docs/concepts/configuration/configmap/)
+file is stored in a github repository, the removal
+needs to be committed to the repository to finalize the removal.
+Finally, the `migrate` command adds an `inventory`
+section to the Kptfile if it did not already exist. Updates to
+the package can now be applied using `kpt live apply <PACKAGE DIR>`.
+
+### New (Alpha) Commands
+
+#### Migrate
+
+[kpt live migrate](./migrate)
+
+#### Install Resource Group
+
+[kpt live install-resource-group](./install-resource-group)
+
+### Updated Existing Commands
+
+#### Init
+
+[kpt live init](./init)
+
+### Troubleshooting and Verifying
```

Comment:
Is there an error case when running `kpt live apply` that tells the user to fall back to manual install of CRD? (Related to comment above)

Reply:
OK, I used the `kpt live apply` instead of `kpt live install-resource-group` to illustrate the error. I added the command to  manually install the CRD as the way to fix and verify the error. Please let me know what you think.